app/clients/db/models/law_policy/family.py

```diff
@@ -65,7 +65,7 @@ class Family(Base):
     )
 
     @hybrid_property
-    def family_status(self) -> FamilyStatus:
+    def family_status(self) -> FamilyStatus:  # type: ignore
```

Comment:
Remember these files are copied from the backend - so need making any chanages in that repo

Reply:
Noted - will do this now!

Reply:
https://github.com/climatepolicyradar/navigator-backend/pull/180